miner/proposal_builder.go

```diff
@@ -139,6 +141,12 @@ func WithMinGoodAtxPct(pct int) Opt {
 	}
 }
 
+func WithEmitEmptyActiveSet(lid types.LayerID) Opt {
```

Comment:
i still don't see this invoked in node.go. what am i missing?

Reply:
oh sorry :)  i thought you are suggesting to rename it